<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbFrom">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSubject">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Subject</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Message</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teMessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFrom">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>20</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTo">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>50</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSubject">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>80</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSend">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
